lib/Sema/CSGen.cpp

```diff
@@ -1750,6 +1750,24 @@ namespace {
           return nullptr;
         }
 
+        if (numElements == 1 && CS.isForCodeCompletion()) {
+          // If a single-element array has a dictionary contextual type, pretend
+          // it was parsed as a dictionary literal for code completion purposes,
+          // as it is likely the value has just not been written yet.
+          CS.getASTContext();
```

Comment:
I guess this is a leftover from some debugging...

Reply:
😅